spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -386,7 +386,7 @@ class Array
           def foo_int; end
         end
 
-        class Module
+        class Range
```

Comment:
Why did these turn into `Range`?

Reply:
From the commit comment: 

> We sometimes patch `Module` inside Tapioca so it is not a good candidate for testing this behaviour.

I guess we say the autoload hook appearing in the test output.

Reply:
Since we are prepending a module inside of `Module` in `AutoloadTracker`, this test started picking up that new module. So I opted to go for a class that we don't modify inside of Tapioca.